monitor/events/list-get-example-actorsid-resourcesid-error/list-get-example-actorsid-resourcesid-error.2.x.js

```diff
@@ -11,6 +11,6 @@ var params = {
 };
 client.events.list(params, function(error, events) {
     for (var e in events) {
-        process.stdout.write(e.description);
+        process.stdout.write(e);
```

Comment:
@hortega I think we're not supporting old versions like this, also, I saw you deleted an old version, so what's is the approach for these cases?